BEIMA.Backend.FT/FunctionalTestBase.cs

```diff
@@ -29,6 +34,21 @@ public void BaseOneTimeSetUp()
             // Uncomment for Azure server:
             // BeimaClient Client = new BeimaClient("https://beima-service.azurewebsites.net");
             TestClient = new BeimaClient("http://localhost:7071");
+
+            // Have at least one admin user in the DB at all times as tests wil need it for auth
+            var initialUserList = await TestClient.GetUserList();
```

Comment:
This isn't going to work because GetUserList and AddUser are going to need to require a logged in admin user to even use (once we get that implemented). This might be ok for now but we will need another solution when we put authentication on the user endpoints.

Reply:
Unless of course we modify the user endpoints to not check for authentication until an admin user exists in the database. Was that what you were planning?

Reply:
I think for the testing side of things, we might end up doing a raw-mongo connecter call to add a user in. I think this will be ok just because for the live application there will need to be at least on user in the db as you'll have to login before doing anything. Keelan, Ashlyn, and I were kinda discussing this issue and possible solutions, and were gonna bring it up next scrum.